Formula/ibex.rb

```diff
@@ -12,37 +24,82 @@ class Ibex < Formula
     sha256 "41327b6a0da9b8b2ed888c3c2c33d5f6ff061ea13eceb4b9e8b6e660049d624d" => :mountain_lion
   end
 
-  option "with-java", "Build Java bindings for Choco solver."
+  option "with-java", "Enable Java bindings for CHOCO solver."
+  option "with-affine", "Use Affine Arithmetic plugin"
```

Comment:
Of course, I can add by default the commonly used options (`--with-affine` and `--with-optim` are set in Ibex CI system). Is it better to just remove these two options, or to replace them with their `--without-**` counterpart?
I can also remove the `--with-debug` option, which is maybe useless in brew.